Doc/using/ios.rst

```diff
@@ -33,6 +33,17 @@ running, so you only need to deal with the Python code itself.
 Python at runtime on iOS
 ========================
 
+iOS version compatibility
+-------------------------
+
+The minimum supported iOS version is specified at compile time, using the
+:option:`--host` option to ``configure``. By default, when compiled for iOS,
+Python will be compiled with a minimum supported iOS version of 13.0. To use a
+different miniumum iOS version, provide the version number as part of the
+:option:`--host` argument - for example,
```

Comment:
We try to only link once to the same reference in a paragraph, in order to reduce link density in the docs.

```suggestion
:option:`!--host` argument - for example,
```